README.md

```diff
@@ -59,7 +59,7 @@ Include the following zendesk package version in your `packages.yml` file:
 ```yml
 packages:
   - package: fivetran/zendesk
-    version: [">=0.10.0", "<0.11.0"]
+    version: [">=0.11.0", "<0.12.0"]
```

Comment:
also gotta update the source package in the dependency matrix

Reply:
Good catch, will update!

Reply:
Updated